pkg/azuredisk/azuredisk_v2.go

```diff
@@ -118,6 +118,7 @@ type DriverV2 struct {
 	azdiskClient         azdisk.Interface
 	azDriverNodeInformer azdiskinformertypes.AzDriverNodeInformer
 	deviceChecker        *deviceChecker
+	enableMountReplicas  bool
```

Comment:
Since the configuration is referenced in the `config` field above, there's no need to add an additional field for this setting here.